test/utils/print/OtherIndividualPrintHelperSpec.scala

```diff
@@ -126,8 +126,6 @@ class OtherIndividualPrintHelperSpec extends SpecBase {
             AnswerRow(label = messages("otherIndividual.passportOrIdCardDetailsYesNo.checkYourAnswersLabel", name.displayName), answer = Html("Yes"), changeUrl = amendRts.PassportOrIdCardDetailsYesNoController.onPageLoad(mode).url),
             AnswerRow(label = messages("otherIndividual.passportOrIdCardDetails.checkYourAnswersLabel", name.displayName), answer = Html("United Kingdom<br />Number ending C456<br />10 October 2030"), changeUrl = amendRts.PassportOrIdCardDetailsController.onPageLoad(mode).url),
             AnswerRow(label = messages("otherIndividual.mentalCapacityYesNo.checkYourAnswersLabel", name.displayName), answer = Html("Yes"), changeUrl = rts.MentalCapacityYesNoController.onPageLoad(mode).url),
```

Comment:
Trailing comma can be removed. Otherwise happy with the changes

Reply:
Removed the trailing comma